CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 - Added support for VOLATILE/IMMUTABLE keyword when registering UDFs.
 - Added support for specifying clustering keys when saving dataframes using `DataFrame.save_as_table`.
 - Accept `Iterable` objects input for `schema` when creating dataframes using `Session.create_dataframe`.
+- Added the property `DataFrame.session` to return a session object.
+- Added the property `Session.session_id` to return an integer that represents session id.
+- Added the property `Session.connection` to return a ServerConnection object .
```

Comment:
I don't think we need "the property" given it is implied in the method name. What do you think?

Reply:
I think we could keep it here, it is more friendly to those who are not familiar with Python.

Reply:
This isn't accurate. The returned connection is a `SnowflakeConnection`.
The `ServerConnection` isn't exposed.

Reply:
will modify it